_data/toc.yaml

```
@@ -1182,8 +1182,32 @@ manuals:
               title: Edge releases
             - path: /desktop/windows/release-notes/archive/
               title: Older versions
-    - path: /desktop/linux/
-      title: Linux (Beta)
+    - sectiontitle: Linux
+      section:
+        - path: /desktop/linux/install/
+          title: Install Docker Desktop for Linux
+        - sectiontitle: Installation per distro
+          section:
+            - path: /desktop/linux/install/debian/
+              title: Install on Debian
+            - path: /desktop/linux/install/fedora/
+              title: Install on Fedora
+            - path: /desktop/linux/install/ubuntu/
+              title: Install on Ubuntu
+            - path: /desktop/linux/install/archlinux
+              title: Install on Arch
+        - path: /desktop/linux/desktop-vs-engine
```

Missing a trailing `/` (looks like that breaks the navigation)

```suggestion
        - path: /desktop/linux/desktop-vs-engine/
```